w07h02/test/pgdp/security/TrackTest.java

```diff
@@ -167,13 +167,28 @@ void assertSetRange(String type, boolean up, int start, int end, int amountOfPos
     @Test
     void setRangeNotCircle() {
         assertSetRange("down", true, 0, 1, 3, List.of(0, 1));
+        assertSetRange("down", true, 0, 2, 3, List.of(0, 1, 2));
+        assertSetRange("down", true, 5, 10, 20, List.of(5, 6, 7, 8, 9, 10));
     }
 
     @Test
     void setRangeCircle() {
         assertSetRange("down", true, 2, 1, 3, List.of(2, 0, 1));
+        assertSetRange("down", true, 3, 2, 5, List.of(3, 4, 5, 0, 1, 2));
+        assertSetRange("down", true, 1, 1, 3, List.of(1));
     }
 
+    @Test
+    void setRangeInvalid() {
```

Comment:
Could you remove this test? "die int-Parameter innerhalb der Array-Grenzen sind ( >= 0 und < posts.length)"

Reply:
ok